cirq/google/ops/calibration_tag.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A class that can be used to denote a physical Z gate."""
```

Comment:
Need to update module docstring, or just remove since the class docstring should be sufficient.

For that matter, having separate files for each tag might be a bit overkill; perhaps we could put both physical z tag and calibration tag in a single `tags.py` file. Up to you though.